docs/zh/latest/getting-started.md

```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
 #
 -->
 
-## 概述
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-本文是 Apache APISIX 的快速入门指南。快速入门分为三个步骤：
+本文将为你介绍 Apache APISIX 的概念、功能以及如何使用 APISIX。
 
-1. 通过 [Docker Compose](https://docs.docker.com/compose/) 安装 Apache APISIX。
-1. 创建路由并绑定上游。
-1. 使用命令行语句 `curl` 验证绑定之后返回的结果是否符合预期。
+通过本文你可以了解到以下内容：
 
-除此之外，本文也提供了 Apache APISIX 的一些进阶操作技巧，包括：添加身份验证、为路由添加前缀、使用 APISIX Dashboard 以及常见问题排查。
+- Apache APISIX 是什么？
+- APISIX 的架构及主要概念。
+- 如何使用 Docker 安装并运行 APISIX。
+- 如何使用 Admin API 创建第一个路由并配置上游。
+- 如何使用 APISIX Dashboard。
+- 如何寻求帮助。
 
-我们将以下面的 `echo` 端点为例，它将返回我们传递的参数。
+## Apache APISIX 是什么？
 
-**请求内容**
+Apache APISIX 是一个开源的云原生 API 网关，作为 API 网关，它兼具动态、实时、高性能等特点，提供了负载均衡、动态上游、灰度发布、服务熔断、身份认证、可观测性等丰富的流量管理功能。你可以使用 Apache APISIX 来处理传统的南北向流量，以及服务间的东西向流量，也可以当做 K8s Ingress controller 来使用。
+
+### APISIX 的功能
+
+- 多平台支持：APISIX 提供了多平台解决方案，它不但支持裸机运行，也支持在 Kubernetes 中使用，还支持与 AWS Lambda、Azure Function、Lua 函数和 Apache OpenWhisk 等云服务集成。
+- 全动态能力：APISIX 支持热加载，这意味着你不需要重启服务就可以更新 APISIX 的配置。
+- 精细化路由：APISIX 支持使用 NGINX 所有的内置变量做为路由的条件，你可以自定义匹配函数来过滤路由。
+- 运维友好：APISIX 因其对 DevOps 团队的操作友好而闻名。它可以与以下工具和平台集成：[HashiCorp Vault](./plugins/jwt-auth.md#usage-with-hashicorp-vault)、[Zipkin](./plugins/zipkin.md)、[Apache SkyWalking](./plugins/skywalking.md)、[Consul](./discovery/consul_kv.md)、[Nacos](./discovery/nacos.md)、[Eureka](./discovery.md)。通过 [APISIX Dashboard](/docs/dashboard/USER_GUIDE)，运维人员可以通过友好且直观的 UI 配置 APISIX。
+- 多语言插件支持：APISIX 支持多种开发语言进行插件开发，开发人员可以选择擅长语言的 SDK 开发自定义插件。
+
+## 主要概念
+
+下图为 APISIX 的架构：
+
+![flow-software-architecture](https://raw.githubusercontent.com/apache/apisix/master/docs/assets/images/flow-software-architecture.png)
+
+下表是本文涉及到的 APISIX 的主要概念和组件：
+
+| 概念/组件    | 描述                                                                                             |
+|-------------|--------------------------------------------------------------------------------------------------|
+| Route       | 通过路由定义规则来匹配客户端请求，根据匹配结果加载并执行相应的插件，最后把请求转发给到指定的上游应用。  |
+| Upstream    | 上游的作用是按照配置规则对服务节点进行负载均衡，它的地址信息可以直接配置到路由或服务上。               |
+| Admin API   | 用户通过该组件可以控制 APISIX 实例的 API。                                                         |
+
+## 前提条件
+
+在开始使用 APISIX 之前，请确保你已经安装以下应用：
+
+- [Docker](https://www.docker.com/) 和 [Docker Compose](https://docs.docker.com/compose/)。
+- [curl](https://curl.se/docs/manpage.html) 用于测试 API。你也可以使用 [Hoppscotch](https://hoppscotch.io/) 之类的工具。
+- 本文使用的上游服务是 [httpbin.org](https://httpbin.org)，你可以使用它进行测试。它是一个返回服务，这意味着它将返回我们在请求中传递的参数。
```

Comment:
How about converting this NOTE to the `INFO` mark? (Not in this PR) AFAIK, some users don't know what's `httpbin` is and they will ignore this part usually 🤣

cc @hf400159 @navendu-pottekkat